iothub/device/src/Common/Api/ClientApiVersionHelper.cs

```diff
@@ -8,7 +8,15 @@ class ClientApiVersionHelper
         const string ApiVersionQueryPrefix = "api-version=";
         const string ApiVersionNov2016 = "2016-11-14";
         const string ApiVersionJune2017 = "2017-06-30";
-        public const string ApiVersionString = ApiVersionJune2017;
+
+#if ENABLE_MODULES_SDK
+        const string ApiVersionEdgePublicPreview = "2017-11-08-preview";
```

Comment:
>ApiVersionEdgePublicPreview [](start = 21, length = 27)

Why can't this be the latest regardless of the #if flag? This is the modules_repo anyhow..